migreringer/src/main/resources/db/migration/defaultDS/3.1/V3.1_01__TFP-5571-lokalt_oppgave_tabelle.sql

```diff
@@ -3,14 +3,16 @@ create table OPPGAVE
     ID NUMBER(19) not null
         constraint PK_OPPGAVE
             primary key,
-    OPPGAVE_TYPE VARCHAR2(20 char) not null,
-    JOURNALPOST_ID VARCHAR2(32 char) not null,
+    TYPE VARCHAR2(20 char) not null,
+    STATUS VARCHAR2(20 char) not null,
+    JOURNALPOST_ID VARCHAR2(32 char),
     BRUKER_ID VARCHAR2(19 char),
     BEHANDLING_TEMA VARCHAR2(20 char),
     PRIORITET VARCHAR2(20 char),
     BESKRIVELSE varchar2(200 char),
     ENHET VARCHAR2(10 char),
     FRIST TIMESTAMP(3) not null,
+    RESERVERT_AV VARCHAR2(20 char),
     VERSJON INTEGER default 0 not null,
```

Comment:
Hvorfor tenker du vi har behov for versjonering her?

Reply:
Det er en standard ting for Optimistisk Concurrency Control - annoteres med @Version i entitet.
Her kan potensielt sett flere prøve  å jobbe med journalpost samtidig.
Kommer litt an på om man setter ferdig eller sletter. Greit å bruke ferdigstilt for debugging

Reply:
Ja det er mer et teknisk parameter som hibernate bruker til å hindre to eller flere prosesser som prøver å oppdatere samme objekt samtidig. Jeg tenkte ikke å slette med ferdigstille. Mulig vi rydder opp med batch eller prosess task etterpå ved behov.